web_responsive/static/src/components/control_panel/control_panel.scss

```diff
@@ -18,16 +18,6 @@
         }
     }
     // For FULL HD devices
-    @media (min-width: 1900px) {
-        .o_cp_top_left,
```

Comment:
As @pedrobaeza wrote it's better to avoid using this rule in popup rather than removing it at all.
You can change this part of code like this:
```scss
            .o_action_manager & .o_cp_top_left,
            .o_action_manager & .o_cp_bottom_left {
                width: 40%;
            }
            .o_action_manager & .o_cp_top_right,
            .o_action_manager & .o_cp_bottom_right {
                width: 60%;
            }
```

Reply:
I’ve tested the solution, it looks like it works. I’ve kept the original width (the comment below was inverting them, I do not think that was intentional).